src/worker.ts

```diff
@@ -185,7 +185,7 @@ export function makeNewWorker(
         await withPgClient(client =>
           client.query({
             text: `SELECT FROM ${escapedWorkerSchema}.fail_job($1, $2, $3);`,
-            values: [workerId, job.id, message],
+            values: [workerId, job.id, message || ''],
```

Comment:
Thanks for spotting this. Let's fix it in the `logger.error` call too. I can't suggest the change up there, but here's the logic I'm after

```ts
const { message: rawMessage, stack } = err;
const message = rawMessage || String(err) || 'Non-error thrown (no message)';
```

Then replace `err.message` with `message`.